api/src/wfl/service/dockstore.clj

```diff
@@ -0,0 +1,15 @@
+(ns wfl.service.dockstore
+  "Wrappers for Dockstore HTTP APIs."
+  (:require [clj-http.client   :as http]
+            [clojure.string    :as str]
+            [wfl.environment   :as env]
+            [wfl.util          :as util]))
+
+(defn ^:private dockstore-url [& parts]
+  (let [url (util/de-slashify (env/getenv "WFL_DOCKSTORE_URL"))]
+    (str/join "/" (cons url parts))))
```

Comment:
I don't get the `de-slashify`, but OK.

Reply:
If you specify the URL ending with a `/` in the environment variable.

Reply:
Obviously not a big deal, but just don't do that?
Who else is specifying our environment variables?

Reply:
It's an easy mistake to make I guess in testing/overriding for system tests. At least this way it doesn't matter if someone specifies a url with a trailing `/`.